equinix/resource_ecx_l2_connection.go

```diff
@@ -585,42 +614,21 @@ func resourceECXL2ConnectionCreate(ctx context.Context, d *schema.ResourceData,
 		return diag.FromErr(err)
 	}
 	d.SetId(ecx.StringValue(primaryID))
```

Comment:
> To handle that without a full scan I propose to keep RedundantUUID but without using the value returned from the API. Instead value will be set in _[ID and RedundantUUID, set during creation]_

We could store `primaryID:secondaryID` as the ID. Do you think that would help?

On new connections, we would have in the ID all of the data to refresh the state of the resource.

Although I can't think of any scenarios off the top of my head, it seems like relying on any other state, such as`RedundantUUID`, for refreshes could be risky.

Reply:
My concern is that users in terraform are used to the `.id` to reference resources and not the `.uuid` so a change like that would put more complexity/responsibilities on the user side. It would also introduce a breaking change, while using RedundantUUID users will no experiment any change

Reply:
I wasn't considering use of `.id`. That's a good reason not to change things.